man/redfishpower.8.in

```diff
@@ -62,8 +62,12 @@ Set default path and optional post data to turn off plug.
 .TP
 .I "setplugs plugnames hostindices"
 Associate a plug name with one of the hostnames specified on the
-command line, referred to by its zero origin index.  Can be called
-multiple times to configure all possible plugs.
+command line, referred to by its zero origin index. Can be called
+multiple times to configure all possible plugs.  In most
+cases the number of plugs should equal the number of indices.
+Multiple plugs can be mapped to a single host index if used with plug
+name substitution.  The plug name can be substituted into the URI path
+by specifying "{{plug}}" when configuring the path.
```

Comment:
The first sentence uses plug (singular) but `plugnames` can represent multiple plugs.  Maybe it should say
> Map one or more plug names to one or more hostnames specified on the command line.  Hostnames are referenced by their zero-origin list index.  For example: "setplugs p[0-7] [0-7]" assigns p0 to the first hostname, p1 to the second, etc..

On the new part, maybe instead of "in most cases" which sounds like a value judgement you could just say:
> In some cases, multiple plugs may be mapped to the same hostname, for example when a blade chassis with one network address contains multiple slots, where a plug will represent just one slot in the chassis.

Reply:
Shouldn't the "{{plug}}" substitution be described with the `setpath` command?

Reply:
> Shouldn't the "{{plug}}" substitution be described with the setpath command?

I went back on forth on this.  Plug substitution only matters when you set > 1 plug to 1 host index, so I felt it belonged more in the `setplugs` documentation.

I guess I could add info on it in both areas.

Reply:
Or you could look at it as simply substutitiong something in the path, and whether or not you eventually end up with uniqe plug -> (host, path) mappings is a higher level problem.

Reply:
> Or you could look at it as simply substutitiong something in the path, and whether or not you eventually end up with uniqe plug -> (host, path) mappings is a higher level problem.

(Related to other conversation regarding the plug-sub-required flag) I initially disliked the fact that if a user messes up the config, like `pm --off foo-blade0` would actually end up powering off the entire chassis.  Which is why I sort of put them all under the `setplugs` area (and have the `plug-sub-required` flag).

But maybe it's a higher level problem I shouldn't be concerned about.  Like you say, I should just mention "{{plug}}" under `setpath` and remove the "plugsub required" flag.  If the user messes up .. the user messes up?

Reply:
Yeah I'm ok with that.  Where user is the dev script developer.